src/hooks/users.ts

```diff
@@ -50,21 +55,8 @@ export const useUsersById = () => {
   return useSelector(usersByIdSelector);
 };
 
-export const useVenueUsers = (): readonly WithId<User>[] =>
-  useSelector(usersSelector) ?? [];
-
-export const useCampPartygoers = (
-  venueName: string
-): readonly WithId<User>[] => {
-  const partygoers = usePartygoers();
-
-  const lastSeenThreshold = useUserLastSeenThreshold();
+export const useVenueUsers = (): readonly WithId<User>[] => {
+  useConnectUsers();
 
-  return useMemo(
-    () =>
-      partygoers.filter(
-        (partygoer) => partygoer?.lastSeenIn?.[venueName] > lastSeenThreshold
-      ),
-    [partygoers, lastSeenThreshold, venueName]
-  );
+  return useSelector(usersSelector) ?? [];
```

Comment:
Because the old version of `useCampPartygoers` called `usePartygoers` it had the time based 'recent' filtering logic in it. Yet now `useVenueUsers` just selects directly from `usersSelector`.

I believe there were some refactors made in these 6 commits since my last review that updated a number of things to use `useVenueUsers` that used to use `useCampPartygoers`, which means those places will lose the 'recent user filtering' aspect, which may have negative impacts on counting/similar depending which files they were used in.

Reply:
We still have `usePartygoers`. `useCampPartygoers` was replaced with `usePartygoers`, which has time-based filtering

Reply:
Ah ok. Sounds good :)